docs/designs/033-grpc-sink-connector.md

```
@@ -0,0 +1,96 @@
+service PlatformConnector {
+  rpc HealthEventOccurredV1(HealthEvents) returns (google.protobuf.Empty) {}
+}
+```
+
+No new proto definitions are needed. The external server receives the same
+structured data that the store and K8s connectors receive — full fidelity,
+no truncation.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)